app/src/common/shared/org/mozilla/vrbrowser/VRBrowserActivity.java

```diff
@@ -1227,36 +1230,36 @@ public void updateVisibleWidgets() {
     }
 
     @Override
-    public void startWidgetResize(final Widget aWidget, float aMaxWidth, float aMaxHeight, float minWidth, float minHeight) {
+    public void startWidgetResize(@NonNull final Widget aWidget, float aMaxWidth, float aMaxHeight, float minWidth, float minHeight) {
         mWindows.enterResizeMode();
         queueRunnable(() -> startWidgetResizeNative(aWidget.getHandle(), aMaxWidth, aMaxHeight, minWidth, minHeight));
     }
 
     @Override
-    public void finishWidgetResize(final Widget aWidget) {
+    public void finishWidgetResize(@NonNull final Widget aWidget) {
```

Comment:
Should never be null but could potentially be null when called from: https://github.com/MozillaReality/FirefoxReality/blob/2516a84dcc5b425a9178dc3f441938ce57e8d2b6/app/src/common/shared/org/mozilla/vrbrowser/ui/widgets/NavigationBarWidget.java#L985

Reply:
Again, base class has it set to `@NonNull`